bot/exts/code_jams/_views.py

```diff
@@ -134,3 +135,118 @@ async def interaction_check(self, interaction: discord.Interaction) -> bool:
             )
             return False
         return True
+
+
+class AddNoteModal(discord.ui.Modal, title="Add a Note for a Code Jam Participant"):
+    """A simple modal to add a note to a Jam participant."""
+
+    def __init__(self, member: discord.Member, mgmt_client: APIClient):
+        super().__init__()
+        self.member = member
+        self.mgmt_client = mgmt_client
+
+    note = discord.ui.TextInput(
+        label="Note",
+        placeholder="Your note..."
+    )
+
+    async def on_submit(self, interaction: discord.Interaction) -> None:
+        """Discord.py default to handle modal submission."""
+        try:
+            user = await self.mgmt_client.get(f"users/{quote_url(str(self.member.id))}/current_team",
```

Comment:
`quote_url` seems unnecessary considering it will always just be the characters 0-9, which don't require any special escapes.
```suggestion
            user = await self.mgmt_client.get(f"users/{self.member.id}/current_team",
```